Bricks/Chapel-Data-Structures/v0.1.0.toml

```diff
@@ -0,0 +1,5 @@
+[brick]
+name = "Chapel Data Structures"
+version = "0.1.0"
+author = ["Louis Jenkins <LouisJenkinsCS@hotmail.com>"]
```

Comment:
This might just need to be a string. Not sure what happens when you put it in brackets...

Reply:
Done.